pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -129,6 +129,9 @@ type GardenletControllerConfiguration struct {
 	// BackupEntry defines the configuration of the BackupEntry controller.
 	// +optional
 	BackupEntry *BackupEntryControllerConfiguration `json:"backupEntry"`
+	// Bastion defines the configuration of the Bastion controller.
+	// +optional
+	Bastion *BastionControllerConfiguration `json:"bastion"`
```

Comment:
Please extend https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardenlet.yaml and https://github.com/gardener/gardener/blob/master/charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml